### DISCUSSION TITLE
Quarto CLI documentation in quarto.org

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#4967] [Feature Requests]
The website is great regarding markdown typesetting.
In addition to what is already in it, I think it would be really great to also include Quarto CLI commands documentation as some kind of reference page (and/or "man" page).

Also, I noticed the `quarto use` command is not at all documented, not even in Quarto CLI itself.
<img width="1124" alt="image" src="https://user-images.githubusercontent.com/8896044/227738301-58da899b-4ecb-49fa-95f8-25adaa907bcd.png">

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/8896044/227738347-0e25d47e-44a3-422c-a918-fc2fc2736978.png">

**Comment by andrewpbray** [comment]
This would be super helpful!

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/598
